src/main/kotlin/org/jitsi/rtp/rtp/RtpPacket.kt

```diff
@@ -26,6 +26,7 @@ import org.jitsi.rtp.util.BufferPool
 import org.jitsi.rtp.util.getByteAsInt
 import org.jitsi.rtp.util.isPadding
 import org.jitsi.utils.observableWhenChanged
+import sun.plugin.dom.exception.InvalidStateException
```

Comment:
Ops, I think that's the wrong class

Reply:
Yeah, Java calls them "Illegal" not "Invalid" (which is stupid because no one's going to arrest you).

Reply:
Oops, indeed.  Thanks.

Reply:
It's `IllegalStateException`, I always seem to want to use `InvalidStateException` for some reason.